index.html

```diff
@@ -62,17 +62,17 @@ <h2>Let's Connect</h2>
     <div class="projects-card">
       <div class="card-header">
           <div class="projects">
-            <img src="./Images/Snapshoot Portfolio__.png" alt="" class="project-screenshot">
+            <img src="./Images/Snapshoot-Portfolio.png" alt="Project One" class="project-screenshot">
```

Comment:
Kindly check your image path, your images are not displaying 😊

![image](https://user-images.githubusercontent.com/73854999/200681489-bf95c631-c8a5-4356-9fc2-aa1da9550299.png)